vcd/resource_vcd_network_test.go

```diff
@@ -958,6 +1006,10 @@ resource "vcd_network_isolated" "{{.ResourceName}}" {
     start_address = "{{.StartStaticIpAddress1}}"
     end_address   = "{{.EndStaticIpAddress1}}"
   }
+  metadata = {
+    {{.MetadataKey}} = "{{.MetadataValue}}"
+    key2 = "value2"
```

Comment:
please move `key2 = "value2"` to params/templating in this PR

Reply:
Done 👍